lib/dao.js

```diff
@@ -1876,8 +1886,17 @@ DataAccessObject.prototype.save = function (options, cb) {
   var hookState = {};
 
   if (options.validate === undefined) {
-    options.validate = true;
+    if (Model.settings.validateUpsert === undefined) {
```

Comment:
I believe `PersistedModel.prototype.save` is not affected by `validateUpsert` setting. I think this block of code should belong to `PersistedModel.prototype.updateAttributes`, which was not changed to support `options.validate` yet?